src/test/resources/org/kohsuke/github/GHReleaseTest/wiremock/testCreateReleaseWithNotes/mappings/2-r_h_t_releases.json

```diff
@@ -0,0 +1,52 @@
+{
+  "id": "2e081774-790c-4d7b-9f4b-71bad948a411",
+  "name": "repos_hub4j-test-org_testcreaterelease_releases",
+  "request": {
+    "url": "/repos/hub4j-test-org/testCreateRelease/releases",
+    "method": "POST",
+    "headers": {
+      "Accept": {
+        "equalTo": "application/vnd.github.v3+json"
+      }
+    },
+    "bodyPatterns" : [
+      {
+        "matchesJsonPath" : "$.[?(@.generate_release_notes  == true)]"
```

Comment:
this is where the newly added parameter is verified.

Pre-existing behavior of not passing `generate_release_notes` when not set is validated in [`createSimpleRelease`](https://github.com/hub4j/github-api/blob/main/src/test/resources/org/kohsuke/github/ReleaseTest/wiremock/testCreateSimpleRelease/mappings/repos_hub4j-test-org_testcreaterelease_releases-3.json#L12-L18) test.